### PR TITLE
Adaptive Icon Support

### PR DESCRIPTION
**Pull request by kdwoo1217**
#996 I think he want to like this called Adaptive Icons on Android Oreo.
I don't know this is right PR.
I read your Release Process in here [https://github.com/zxing/zxing/wiki/Getting-Started-Developing](url).
But I don't know what it means....
If this PR is wrong the send me E-Mail or comment here. Thank you.

screenshot_20180614-005911
screenshot_20180614-005928
screenshot_20180614-010001

### CODE REVIEW

File: android/AndroidManifest.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (srowen) [inline android/AndroidManifest.xml]: This part of the change should still be reverted.

File: android/res/values-fa/strings.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (srowen) [inline android/res/values-fa/strings.xml]: Likewise this

File: android/res/xml/preferences.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (srowen) [inline android/res/xml/preferences.xml]: And this

**Comment by kdwoo1217** [comment]
Adaptive icon is show like this on Android 8.0.
adaptive-icons

**Pull request closed without merging**